cmd/crc/cmd/root.go

```diff
@@ -102,7 +103,7 @@ func checkIfMachineMissing(client machine.Client) error {
 		return err
 	}
 	if !exists {
-		return fmt.Errorf("Machine '%s' does not exist. Use 'crc start' to create it", client.GetName())
+		return crcErr.VMNotExist
 	}
```

Comment:
Just a note for future improvement, `client.Exists()` could return a `crcErr.VMNotExist` when the VM does not exist, which would make `checkIfMachineMissing()` implementation simpler/shorter.